src/app/pages/contacts/contacts.component.html

```diff
@@ -1 +1,3 @@
-<p>contacts works!</p>
+<app-layout bgColor="blue">
```

Comment:
do you really need the component for each page, try to re-write using only one and @Input directive